myhpom/static/myhpom/combodate-1.0.7/README.md

```diff
@@ -0,0 +1,25 @@
+# Combodate
+Dropdown date and time picker.  
+This plugin converts `<input type="text">` into dropdown elements to pick day, month, year, hour, minutes and seconds.
+
+* support of date, time, datetime
+* 12h / 24h format
+* based on [moment.js](http://momentjs.com)
+
+
+## Demo, Docs and Download
+**http://vitalets.github.com/combodate**
+
+
+## Installation
+````
+bower install combodate
```

Comment:
Ideally we don't have 3rd party libraries checked into the repository...seeing that this wasn't in npm and can't be included there I thought yesterday, "I guess we don't have any choice." But...then I saw this bower install bit. We could/should use it to install rather than checking this in. I'd suggest adding bower as a dev dependency (I can do this cleanup bit in my ticket if you'd rather punt - I realize hacking docker & setting up bower & static includes isn't the funnest).

Reply:
It looks like adding the following to package.json dependencies will do the trick:
    "combodate": "vitalets/combodate#1.1.0"
and no need for bower.

Reply:
awesome!